docs: typos in production checklist (#5155)
zitadel/zitadel: docs/docs/self-hosting/manage/productionchecklist.md

### Infrastructure Configuration

> - [ ] Make use of configuration management such as Terraform to provision all of the below

Reviewer: ```suggestion
- [ ] Make use of configuration management tools such as Terraform to provision all of the below
```

> - [ ] Use version control to store the provisioning

Author: @thesephirot ping

> - [ ] Use version control to store the provisioning

Reviewer: ```suggestion
```

- [ ] High Availability for critical infrastructure components (depending on your setup)
  - [ ] Loadbalancer
  - [ ] [Reverse Proxies](https://zitadel.com/docs/self-hosting/manage/reverseproxy/reverse_proxy)
  - [ ] Web Application Firewall

#### Networking
- [ ] Use a Layer 7 Web Application Firewall to secure ZITADEL that supports **[HTTP/2](/docs/self-hosting/manage/http2)**
  - [ ] Secure the access by IP if needed
  - [ ] Secure the access by rate limits for specific endpoints (e.g. API vs frontend) to secure availability on high load. See the [ZITADEL Cloud rate limits](https://zitadel.com/docs/apis/ratelimits) for reference.
  - [ ] Doublecheck your firewall for IPv6 connectivity and change accordingly

### ZITADEL configuration
- [ ] Configure a valid [SMTP Server](/docs/guides/manage/console/instance-settings#smtp) and test emails
- [ ] Add [Custom Branding](/docs/guides/manage/customize/branding) if required
- [ ] Configure a valid [SMS Service](/docs/guides/manage/console/instance-settings#sms) such as Twilio if needed
- [ ] Configure your privacy policy, terms of service and a help Link if needed

> - [ ] Secure your [masterkey](https://zitadel.com/docs/self-hosting/manage/configure)

Reviewer: ```suggestion
- [ ] Keep your [masterkey](https://zitadel.com/docs/self-hosting/manage/configure) in a secure storage
```

- [ ] Declare and apply zitadel configuration using the zitadel terraform [provider](https://github.com/zitadel/terraform-provider-zitadel) 

### Security
- [ ] Use a FQDN and a trusted valid certificate for external [TLS](/docs/self-hosting/manage/tls_modes#http2) connections

> - [ ] Make use of different service accounts to secure ZITADEL within your hyperscaler or Kubernetes 

Reviewer: ```suggestion
- [ ] Create service accounts for application that interacts with ZITADEL's APIs
```

> - [ ] Make use of a CDN service if needed to ease maintainability and firewall/DNS/WAF configuration

Reviewer: ```suggestion
- [ ] Make use of a CDN service to decrease the load for static assets served by ZITADEL
```

> - [ ] Make use of a [security scanner](https://owasp.org/www-community/Vulnerability_Scanning_Tools) to test your application and cluster

Reviewer: ```suggestion
- [ ] Make use of a [security scanner](https://owasp.org/www-community/Vulnerability_Scanning_Tools) to test your application and deployment environment
```
